client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.js

```diff
@@ -0,0 +1,216 @@
+                  onClick={e => {
+                    dispatch({
+                      type: 'REMOVE_USER_FROM_TAS_PROJECT',
+                      payload: {
+                        projectId,
+                        id: el.row.original.username
+                      }
+                    });
+                  }}
+                >
+                  Remove
+                </Button>
+              )}
+            </>
+          );
+        }
+      }
+    ],
+    [rawData, removingUserOperation]
+  );
+  const {
+    getTableProps,
+    getTableBodyProps,
+    rows,
+    prepareRow,
+    headerGroups
+  } = useTable({
+    columns,
+    data
+  });
+  return (
+    <table styleName="manage-team-table" {...getTableProps()}>
+      <thead>
+        {headerGroups.map(headerGroup => (
+          <tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <th {...column.getHeaderProps()}>{column.render('Header')}</th>
+            ))}
+          </tr>
+        ))}
+      </thead>
+      <tbody {...getTableBodyProps()}>
+        {rows.map(row => {
+          prepareRow(row);
+          return (
+            <tr>
+              {row.cells.map(cell => (
+                <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+              ))}
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+  );
+};
+AllocationsManageTeamTable.propTypes = {
+  rawData: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  projectId: PropTypes.number.isRequired
+};
+
+const AllocationsManageTeamModal = ({
+  isOpen,
+  toggle,
+  projectId,
+  projectName
+}) => {
+  const dispatch = useDispatch();
+  const { teams, loadingUsernames, search } = useSelector(
+    state => state.allocations
+  );
+  const addUserOperation = useSelector(
+    state => state.allocations.addUserOperation
+  );
+  const error = addUserOperation.error || search.error;
+  useEffect(() => {
+    dispatch({
+      type: 'ALLOCATION_OPERATION_REMOVE_USER_INIT'
+    });
+  }, [isOpen]);
+
+  const isLoading =
+    loadingUsernames[projectId] && loadingUsernames[projectId].loading;
+
+  const onAdd = useCallback(
+    newUser => {
+      dispatch({
+        type: 'ADD_USER_TO_TAS_PROJECT',
+        payload: {
+          projectId,
+          id: newUser.user.username,
+          projectName
+        }
+      });
+    },
+    [projectId, dispatch]
+  );
+
+  const onChange = useCallback(
+    query => {
+      dispatch({
+        type: 'GET_USERS_FROM_SEARCH',
+        payload: {
+          term: query
+        }
+      });
+    },
+    [dispatch]
+  );
+
+  return (
+    <Modal isOpen={isOpen} toggle={toggle} styleName="root">
+      <ModalHeader>Manage Team</ModalHeader>
+      <ModalBody className="p-2">
+        <div styleName="search-bar-wrapper">
+          {error ? <Message type="error">Something went wrong.</Message> : null}
+          <UserSearchbar
+            members={teams[projectId]}
```

Comment:
there is a bug here and we don't know if we have received the team members yet.  a fix is in !572